src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      name="search-try"
+      action="/submit"
+      method="post"
+      class="search-block"
+      data-qa="big"
```

Comment:
The `data-qa` attribute for the first form is set to `big`, which does not meet the task requirements. Please ensure that the `data-qa` attributes are correctly set as per the task description.